src/contributing/libraries_macros.md

```diff
@@ -0,0 +1,25 @@
+# Libraries
+
+# Macros and attributes
+The hax engine understands only one attribute: `#[_hax::json(PAYLOAD)]`,
+where `PAYLOAD` is a JSON serialization of the Rust enum
+`hax_lib_macros_types::AttrPayload`.
+
+Note `#[_hax::json(PAYLOAD)]` is a [tool
+attribute](https://github.com/rust-lang/rust/issues/66079): an
+attribute that is never expanded.
+
+In the engine, the OCaml module `Attr_payloads` offers an API to query
+attributes easily. The types in crate `hax_lib_macros_types` and
```

Comment:
Could you give an example how to use the API in `Attr_payloads`?

Reply:
Sure, I'm gonna write something now